update-scripts/ensure-not-yet-deployed.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+die () {
+	echo "$*" >&2
+	exit 1
+}
+
+set -x
+architecture=
+while case "$1" in
+--architecture=*)
+	architecture=${1#*=}
+	if test aarch64 = "$architecture"
+	then
+		MINGW_PACKAGE_PREFIX=mingw-w64-clang-aarch64
+	else
+		MINGW_PACKAGE_PREFIX=mingw-w64-$architecture
+	fi
+	;;
+-*) die "Unhandled option: '$1'";;
+*) break;
+esac; do shift; done
+
+test $# = 1 ||
+die "Usage: $0 <directory-containing-PKGBUILD>"
+
+cd "$1" ||
+die "Could not switch to '$1'"
+
+. ./PKGBUILD ||
+die "No/invalid PKGBUILD in '$1'?"
+
+case "$arch,$architecture" in
```

Comment:
Yes, that comes from `PKGBUILD`. See e.g. [here](https://github.com/git-for-windows/MINGW-packages/blob/2a1de13472e0894437a572c5c1e34d3f42e67612/mingw-w64-curl/PKGBUILD#L9). And we source `PKGBUILD` in line 30.

Reply:
Ok, I've missed that we source `PKGBUILD`.

Reply:
Fixed via b8ef362c2cdcc51489298b0cf7c476b27e53a89d (which also has an enhanced commit message: it now mentions why we want to prevent re-deployments).

@rimrul could you have another look, please?